modules/microshift-accessing-cluster-locally.adoc

```diff
@@ -25,7 +25,7 @@ $ mkdir -p ~/.kube/
 +
 [source,terminal]
 ----
-$ sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
+$ sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/
```

Comment:
```suggestion
$ sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
```

Maybe that was lost when resolving the merge conflict?

Reply:
No, actually, that was in response to @ggiguash 's earlier comments that .kube/config was a file and .kube/config the folder. We made the directory just above: $ mkdir -p ~/.kube/
Which should it be, then?

Reply:
We're creating a folder `~/.kube/` and writing a file `~/.kube/config` to it.
So we need to keep the command as is (`... > ~/.kube/config`). I assume @ggiguash was referring to the sentence above, which should read "Copy the generated `kubeconfig` file to `~/.kube/config`."